packages/gp2-components/src/organisms/__tests__/KeywordsModal.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { gp2 as gp2Fixtures } from '@asap-hub/fixtures';
+import { Keyword } from '@asap-hub/model/src/gp2';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ComponentProps } from 'react';
+import { StaticRouter } from 'react-router-dom';
+import KeywordsModal from '../KeywordsModal';
+
+describe('KeywordsModal', () => {
+  const getSaveButton = () => screen.getByRole('button', { name: 'Save' });
+
+  beforeEach(jest.resetAllMocks);
+  type KeywordsModalProps = ComponentProps<typeof KeywordsModal>;
+  const defaultProps: KeywordsModalProps = {
+    ...gp2Fixtures.createUserResponse(),
+    backHref: '',
+    onSave: jest.fn(),
+  };
+
+  const renderModal = (overrides: Partial<KeywordsModalProps> = {}) =>
+    render(
+      <StaticRouter>
+        <KeywordsModal {...defaultProps} {...overrides} />
+      </StaticRouter>,
+    );
+
+  it('renders a dialog with the right title', () => {
+    renderModal();
+    expect(screen.getByRole('dialog')).toContainElement(
+      screen.getByRole('heading', { name: 'Keywords' }),
+    );
+  });
+
+  it.each`
+    keywords        | expected
+    ${undefined}    | ${'Start typing...'}
+    ${['Genetics']} | ${'Genetics'}
+  `('renders keywords with value "$expected"', ({ keywords, expected }) => {
+    renderModal({ keywords });
+    const textbox = screen.getByRole('textbox', {
+      name: /Keywords/i,
+    });
+    expect(textbox).toBeVisible();
+    expect(screen.getByText(expected)).toBeVisible();
+  });
+
+  it('calls onSave with the right arguments', async () => {
+    const onSave = jest.fn();
+    const keywords = ['Genetics'] as Keyword[];
```

Comment:
Instead of importing Keyword directly, use the gp2 import and put "gp2.Keyword[]"

Reply:
I would agree if there were other imports from `gp2`, which is not the case here. So I don't see a point in importing the whole `gp2` if I just need `Keyword`